src/controllers/userController.js

```diff
@@ -0,0 +1,74 @@
+const usersService = require('../services/userService');
+
+const getUsers = (req, res) => {
+  res.send(usersService.getAllusers());
```

Comment:
The function `getAllusers` seems to have a typo in its name. It should be corrected to `getAllUsers` to match typical naming conventions and ensure consistency with other function names.